webrtc-stats/README.md

```diff
@@ -0,0 +1,7 @@
+The following 4 test cases in the `webrtc/` directory test some of the mandatory-to-implement stats defined in WebRTC Statistics:
+
+* `webrtc/getstats.html`
+* `webrtc/RTCPeerConnection-getStats.https.html`
+* `RTCPeerConnection-track-stats.https.html`
+* `RTCRtpReceiver-getStats.https.html`
+* `RTCRtpSender-getStats.https.html`
```

Comment:
Why doesn't all paths start with webrtc/...?

Reply:
They should, but it doesn't matter much - it's a README.